### PR TITLE
Sprint11 david

### PR DESCRIPTION
**Pull request by David-Egan**
Configuration needed to get Celery Beat working. Added instructions to the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
 For issues contact David on Slack
+
+## Configuring your machine for Celery Beat
+How to Use Celery Worker:
+
+1. Make sure amqp is v2.3.2 and Kombu v4.2.1 using 'pip freeze'
+
+2. If on Windows make sure gevent is installed with 'pip freeze'
+
+3. Install RabbitMQ (not actually 100% this is required but I think so)
```
Comment (aaron13michael) [inline README.md]: RabbitMQ is required because you need a message broker. I picked Rabbit for our project.

File: openmic/tasks.py [hunk 2]
```diff
@@ -1,9 +1,58 @@
...
+def test_celery_beat_tasks(): 
+    # only run these if in debug mode. We check with os.environ because if
+    # we are running celery worker we can't use getattr yet
+    if not os.environ["DEBUG"]:
+        pass    
+    create_simple_test_task()
+
+
+@app.task
+def message_performance_subscribers(perf_id):
```
Comment (aaron13michael) [inline openmic/tasks.py]: Shouldn't this task go in the openmic/performanceapi/tasks.py file? Is there a reason you put it in the root of the project?

File: openmic/tasks.py [hunk 3]
```diff
@@ -1,9 +1,58 @@
-from celery import Celery
-import os
+from openmic.celery import app
+import os, time, random, string, logging, json
+from firebase_admin import messaging
+from django_celery_beat.models import CrontabSchedule, PeriodicTask
+import topics
```
Comment (aaron13michael) [inline openmic/tasks.py]: I'm getting an import error from this topics import? What library is it a part of? Have you run into this?

File: openmic/tasks.py (hunk 2 above)
Comment (aaron13michael) [reply openmic/tasks.py]: Might need to set aside some time to help me get this running. Let me know when you're available if you think that'd be good.

File: README.md (hunk 1 above)
Comment (David-Egan) [reply README.md]: 👍

File: openmic/tasks.py (hunk 3 above)
Comment (David-Egan) [reply openmic/tasks.py]: I was using a topics.py file to test it should've been removed.

File: openmic/tasks.py (hunk 2 above)
Comment (David-Egan) [reply openmic/tasks.py]: Moved it over. Do you still need help setting this up?

**Pull request merged**